src/WebJobs.Script.WebHost/Resources/app_offline.htm

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" >
```

Comment:
We need to decide what we want the default offline file content to be. This is just a simple placeholder I generated online.